tests/software/gubbins/main.nf

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GUBBINS } from '../../../software/gubbins/main.nf' addParams( options: [:] )
+
+workflow test_gubbins { 
+    input = file("${launchDir}/tests/data/alignment/test.fas", checkIfExists: true)
```

Comment:
I would suggest adding this file in [this folder](https://github.com/nf-core/modules/tree/master/tests/data/genomics/sarscov2/genome) under `alignment/` maybe? Is that an appropriate name?

https://github.com/nf-core/modules/blob/989fb7d5b7db55d1fa305fad23877f95591f1e1a/tests/software/fastqc/main.nf#L12